src/main/java/com/google/maps/model/OpeningHours.java

```diff
@@ -0,0 +1,58 @@
+package com.google.maps.model;
+
+public class OpeningHours {
+  /**
+   * openNow is a boolean value indicating if the place is open at the current time.
+   */
+  public Boolean openNow;
+
+  static public class Period {
+    static public class OpenClose {
+      public enum DayOfWeek {
+        SUNDAY, MONDAY, TUESDAY, WEDNESDAY, THURSDAY, FRIDAY, SATURDAY,
+
+        /**
+         * Indicates an unknown day of week type returned by the server. The Java Client for Google Maps
+         * Services should be updated to support the new value.
```

Comment:
Also, in all seriousness: what about `null`? Just asking why, I don't have a strong opinion.